dashboard/app/views/devise/shared/_links.haml

```diff
@@ -1,29 +1,46 @@
--# Unless we're already on the Sign in page, show a Sign in link
-- unless ['sessions', 'registrations'].include? controller_name
-  = t('auth.already_signedup')
-  = link_to t('nav.user.signin'), new_session_path(resource_name)
-  %br/
-  %br/
+.devise_link_section
+  .or_break
+    .vertical_bar
+    %p
+      OR
+    .vertical_bar
 
-- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks'
-  = link_to t('auth.need_unlock'), new_unlock_path(resource_name)
-  %br/
-  %br/
+  %div{style: "display: block;"}
+    -# Unless we're already on the Sign in page, show a Sign in link
+    - unless ['sessions', 'registrations'].include? controller_name
+      = t('auth.already_signedup')
+      = link_to t('nav.user.signin'), new_session_path(resource_name)
+      %br/
+      %br/
 
-- if devise_mapping.omniauthable?
-  - if !Rails.env.production?
-    .alert
-      Hi! If you are having trouble with OAuth features on
-      %strong= Rails.env
-      %a{href: 'http://wiki.code.org/display/PROD/Developer+Oauth+setup'}
-        check the wiki.
+    - if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks'
```

Comment:
This is unedited in this PR, from the original file which I think was brought over from https://github.com/plataformatec/devise/blob/master/app/views/devise/shared/_links.html.erb - I'm not entirely against pulling things out, but I'm a bit hesitant to do it here because I don't really know what that code is doing or how to properly test it.